training/tests/integration/conftest.py

```diff
@@ -194,7 +194,30 @@ def gnn_config(
 
     cfg = OmegaConf.merge(template, testing_modifications_with_temp_dir, use_case_modifications)
     OmegaConf.resolve(cfg)
-    return cfg, dataset_urls[0]
+    return cfg
+
+
+@pytest.fixture(
+    params=[
+        ["model=graphtransformer"],
+    ],
+)
+def benchmark_config(
+    testing_modifications_with_temp_dir: OmegaConf,
+    get_tmp_paths: callable,
+) -> tuple[OmegaConf, str]:
+
+    with initialize(version_base=None, config_path="../../src/anemoi/training/config", job_name="benchmark"):
+        template = compose(config_name="config")
```

Comment:
I think you meant to use the graphtransformer, i.e.
```suggestion
        template = compose(config_name="config", overrides=["model=graphtransformer"])
```